lib/zendesk_apps_tools/server.rb

```diff
@@ -17,6 +17,7 @@ class Server < Sinatra::Base
           last_mtime = curr_mtime
         end
       end
+      settings.parameters.merge!(locale: params['locale'])
```

Comment:
Shouldn't locale be a standard parameter for *all* apps.  I think it should be.  I can hardly imagine that an app would have a locale parameter that did not have the same semantics as this locale parameter.